### ISSUE TITLE
Import scratch block in Angular ?

### ISSUE THREAD

**Issue by jeremygcn** [LLK/scratch-blocks#1877]
Can we use scratch block in Angular 7 CLI ? If yes, how ?
Thank's in advance.
Jérémy

**Comment by apple502j** [comment]
This is Node library.

**Comment by jeremygcn** [comment]
Yes but how import scratch node in angular and how the blocks can work ?

**Comment by apple502j** [comment]
I can't guess how, because Scratch does not use Angular. Why don7t you switch to react?

**Comment by MGsniper** [comment]
have no idea. I am stuck too

**Comment by cwillisf** [comment]
Closing as part of an issue cleanup. This was a usage question rather than a bug report, and the codebase has changed significantly since it was filed — scratch-blocks v2 is a complete rewrite built on top of upstream [Blockly](https://github.com/RaspberryPiFoundation/blockly/). Thank you!

If you believe this issue is still relevant to scratch-blocks v2, please let us know and we can reopen it.
